README.md

```diff
@@ -1 +1,93 @@
-# Project2
\ No newline at end of file
+# Project2
+
+Target Audience: Bootcamp students, Bootcamp grads, Junior Developers, and Any developer looking to make some side money.
+
+Full Idea: Platform to connect customers who need dev support on a project to developers who can work on their projects based on their expertise
+
+Features:
+* Personas:
+  * Developer
+    * Sign Up and Create Profile
+      * Enter in Name, Location and Upload Photo OR enter Github Alias and we will pull github Name, Location, and Photo into profile
+      * Answer questionairre on development expertise
+      * Link to developer portfolio
+    * Profile View
+      * Sign-in
+      * View new project matches
+        * Developer has right to refuse a project match if the budget or timeline do not meet their needs
+        * Only once the developer has accepted a project can they reach out to the customer directly
+      * Ratings and reviews from past customers
+      * Projects
+    * Project View
+      * View list of currently assigned projects:
+        * Each project view will contain:
+          * Project name
+          * Task list (remaining and completed)
+            * Once all tasks are completed, the project is closed
```

Comment:
I think implementing the task list may be a little tricky. To me its definitely a nice to have, not so much part o the MVP. We can keep it in there and discuss it further on saturday

Reply:
in terms of closing the project we could just have a simple double check where both the dev and the customer check that the project has been completed. Only once they both agree does the project close

Reply:
What makes it tricky is the fact that it is a list, that to me implies that we would need a separate table in our database that is specific for project tasks. Which is doable but its just one more table we have to associate with everything else. If we make it a simple done column in the project table it eliminates the need for that extra table.

Reply:
Since each project would have a variable number of tasks, it would not be ideal to have 20 columns in our project table  for tasks, plus those tasks require other info such as complete or not, etc. So making an extra table would be the way to go, and then having a foreign key associating tasks with their parent project

Reply:
And i think in terms of querying the database making an extra table would streamline the process since we would have mySQL doing all of the heavy lifting, we would just have to worry about writing the queries